docs/cave/bundles-and-procedures.md

```diff
@@ -1,48 +1,48 @@
-# Bundles and Procedures
-AWIPS contains two methods for saving and loading data resources: **Bundles** are a simple way to save loaded resources to access in future CAVE sessions.  **Procedures** are similar to Bundles, but can be thought of as *groups of bundles* and allows the user to manage saved resources with more control.
+# Displays and Procedures
+AWIPS contains two methods for saving and loading data resources: **Displays** are a simple way to save loaded resources into a bundle file to access in future CAVE sessions.  **Procedures** are similar to Displays, but can be thought of as *groups of displays* and allows the user to manage saved resources with more control.
 
 
-##  Bundles
+##  Displays
 
 ### **File > Load Display**
 
-Load a previously-saved bundle from within the AWIPS system.  The Open Bundle dialog allows you to select your own saved bundles as well as those saved by other users.
+Load a previously-saved display from within the AWIPS system.  The pop-up dialog allows you to select your own saved displays as well as those saved by other users.
 
-Each selected bundle will load its contents to new tabs which are named after the bundle file name (e.g. NAM_ThetaE).
+Each selected display will load its contents to new tabs which are named after the display file name (e.g. GFS_TempWind).
 
-![image](../images/KTZZHoCV5S.gif)
+![image](../images/load-display.gif)
 
-Most saved bundles will consist of a single Map Editor (tab), but with multiple tabs saved each will open again in its own Map Editor.
+Most saved displays will consist of a single Map Editor (tab), but with multiple tabs saved each will open again in its own Map Editor.
 
 ---
 
-### **File > Save Display**
-
-Save a product display within the AWIPS system, synching the bundle between CAVE and the EDEX server.
+### **Load Display from Local Disk**
 
----
+To load a previously-saved display from a path within the file directory of the workstation, select **File > Load Dislay** and then select the **File** button on the right to browse your local directories.
 
-### **File > Manage Bundles**
+![image](../images/load-local-display.gif)
```

Comment:
This gif ends right as you click the okay button?  So you don't get to see what it's like loaded at all?  Just seems like it's cut short?

Reply:
Yeah, I never actually loaded it. Michael's gif was similar - just showed the GUI with local files. I can make it even shorter and not have it look like it presses OK. Just leave it with the open local files

Reply:
It never loads?  so it just sort of spins? does it throw an error?  Or just like nothing at all happens?

Reply:
No I never actually clicked OK to load it.

Reply:
Ahh... Does it take a really long time?  Since we load the data in the first gif, could we load it for this one as well? 
If it takes a while, then yeah, I'd say shorten the video a bit so it doesn't look like you're pressing OK.